### PR TITLE
Small edgeos_config improvements

### PR DESCRIPTION
**Pull request by jplitza**
##### SUMMARY

* The module was littered with magic strings ("set" and "delete") which were too broad a match (because to be commands, they have to be followed by a space).
* The logical flow to handle when commands started with neither "set" nor "delete" is stated more easily using an "else" block, avoiding duplication of the `.startswith()` checks
* Using regular expressions to replace plain strings is overkill

##### ISSUE TYPE

- Bugfix Pull Request

(not really, just cleanup)

##### COMPONENT NAME

edgeos_config

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by felixfontein** [comment]
CC @virtualguy

@jplitza can you add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to)?

**Comment by jplitza** [comment]
> @jplitza can you add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to)?

Done, although I only mentioned the only user-visible change (matching the spaces).

File: changelogs/fragments/199-edgeos-improvements.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+- edgeos_config - match the space after "set" and "delete" commands (https://github.com/ansible-collections/community.network/pull/199)
```
Comment (felixfontein) [inline changelogs/fragments/199-edgeos-improvements.yaml:3]:
```suggestion
- edgeos_config - match the space after ``set`` and ``delete`` commands (https://github.com/ansible-collections/community.network/pull/199).
```

**Review by virtualguy — APPROVED** [review]

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@jplitza thanks for improving the module!
@virtualguy @dericcrago thanks for reviewing and merging!

**Comment by Andersson007** [comment]
@jplitza hi, would you like to have your GH login added to `.github/BOTMETA.yml` as a maintainer of the module?
You'll be notified about Issues/PRs related to the module and your `shipit` will be counted by bot for automerge (needs two for bugfixes and minor changes).
what do you think?

**Comment by jplitza** [comment]
@Andersson007 I certainly know the EdgeOS devices well enough, and probably also the module's code. However, I feel unsure about the whole development ecosystem (`shipit`? Changelog Fragments?), so I'd be glad if you have any pointers in that regard. But in general, yeah, sure, you can put me up for everything that has "edgeos" (or "edgeswitch", for that matter) in its name!

**Comment by Andersson007** [comment]
@jplitza thanks!

Answering your question:
1) We have the bot I mentioned in this collection (as well as in community.general and others) which is tracking the issues/PRs including for keywords such as `shipit`, the full list is [here](https://github.com/ansible-community/collection_bot/blob/main/ISSUE_HELP.md).

2) If your GH login is listed in `.github/BOTMETA.yml` as a maintainer of some stuff and you'll put `shipit` in a PR that touches that stuff, the bot will count your `shipit`. If another maintainer also puts `shipit` (so we need two), the PR will be merged by the bot automatically - so being a maintainer in BOTMETA.yml is a kind of indirect write access.

3) BOTMETA.yml maintainers who are active and provide regular significant contributions (reviews and code contributions) in collection scope can be granted the `write` access to the repository: they can merge others PRs, release the collection, etc.

4) We have the [review checklist](https://github.com/ansible/community-docs/blob/main/review_checklist.rst) and [contributing guide](https://github.com/ansible/community-docs/blob/main/contributing.rst). It's highly recommended to read these documents. Also a maintainer guidelines is being in development.

Thank you!

**Comment by Andersson007** [comment]
@jplitza could you please approve the PR https://github.com/ansible-collections/community.network/pull/256

**Pull request merged**